src/jquery.multiselect.js

```diff
@@ -1,699 +1,887 @@
+
+			return this;
+		},
+
+		isOpen: function () {
+			return this._isOpen;
+		},
+
+		widget: function () {
+			return this.menu;
+		},
+		
+		getButton: function(){
+			return this.button;
+		},
+
+		position: function () {
+			var o = this.options;
+
+			// use the position utility if it exists and options are specifified
+			if ($.ui.position && !$.isEmptyObject(o.position)) {
+				o.position.of = o.position.of || this.button;
+
+				this.menu
+				  .show()
+				  .position(o.position)
+				  .hide();
+
+			// otherwise fallback to custom positioning
+			} else {
+				var pos = this.button.offset();
+
+				this.menu.css({
+					top: pos.top + this.button.outerHeight(),
+					left: pos.left
+				});
+			}
+		},
+
+		collapseOptgroups: function (optgroups) {
+			this.toggleOptgroups(false, optgroups);
+		},
+
+		expandOptgroups: function (optgroups) {
+			this.toggleOptgroups(true, optgroups);
+		},
+
+		toggleOptgroups: function (flag, optgroups) {
+			var $optgroups;
+			if (!optgroups) {
+				$optgroups = this.checkboxContainer.find('li.ui-multiselect-optgroup-label');
+			}
+			else if ($.isArray(optgroups)) {
+				$optgroups = $([]);
+				for (var i = 0, length = optgroups.length; i < length; i++) {
+					$optgroups = $optgroups.add(this.checkboxContainer.find("li.ui-multiselect-optgroup-label span:contains(" + optgroups[i] + ")").closest('li.ui-multiselect-optgroup-label'));
+				}
+			}
+			this._toggleOptgroupCollapse(flag, $optgroups);
+		},
+
+		// react to option changes after initialization
+		_setOption: function (key, value) {
+			var menu = this.menu;
+
+			switch (key) {
+				case 'header':
+					menu.find('div.ui-multiselect-header')[value ? 'show' : 'hide']();
+					break;
+				case 'checkAllText':
+					menu.find('a.ui-multiselect-all span').eq(-1).text(value);
+					break;
+				case 'uncheckAllText':
+					menu.find('a.ui-multiselect-none span').eq(-1).text(value);
+					break;
+				case 'height':
+					menu.find('ul').last().height(parseInt(value, 10));
+					break;
+				case 'minWidth':
+					this.options[key] = parseInt(value, 10);
+					this._setButtonWidth();
+					this._setMenuWidth();
+					break;
+				case 'selectedText':
+				case 'selectedList':
+				case 'noneSelectedText':
+					this.options[key] = value; // these all needs to update immediately for the update() call
+					this.update();
+					break;
+				case 'classes':
+					menu.add(this.button).removeClass(this.options.classes).addClass(value);
+					break;
+				case 'multiple':
+					menu.toggleClass('ui-multiselect-single', !value);
+					this.options.multiple = value;
+					this.element[0].multiple = value;
+					this.refresh();
+					break;
+				case 'position':
+					this.position();
+					break;
+				case 'optGroupCollapsible':
+					checkboxContainer.toggleClass('ui-multiselect-optgroup-collapsible', value);
+					break;
+				case 'optGroupSelectable':
+					checkboxContainer.toggleClass('ui-multiselect-optgroup-selectable', value);

```

Comment:
I think checkboxContainer should be either this.checkboxContainer  or you have to declare var in the method 
var cbContainer = this.checkboxContainer; 
and in case statement
cbContainer.toggleClass('ui-multiselect-optgroup-collapsible', value);

Correct me if I am wrong.